InternetShop/InternetShop/Controllers/ProductController.cs

```diff
@@ -29,13 +29,21 @@ public async Task<IEnumerable<ProductViewModel>> GetAll(CancellationToken cancel
         }
 
         [HttpGet("{id}")]
-        public async Task<ProductViewModel?> GetById(int id, CancellationToken cancellationToken)
+        public async Task<ProductViewModel?> GetById([FromRoute] int id, CancellationToken cancellationToken)
         {
             var product = await _productService.GetById(id, cancellationToken);
             var mappedProduct = _mapper.Map<ProductModel>(product);
             return _mapper.Map<ProductViewModel>(mappedProduct);
         }
 
+        [HttpGet("search")]
+        public async Task<IEnumerable<ProductViewModel>> GetByPrice([FromQuery] decimal price, CancellationToken cancellationToken)
```

Comment:
[FromQuery] decimal price. is not correct. Because we don't want to send price via query.